internal/notifier/notifier_test.go

```diff
@@ -454,9 +455,9 @@ func TestNotifier(t *testing.T) {
 			}()
 		}
 
-		close(shutdown)
-		wg.Wait()
-		notifier.Stop()
```

Comment:
Notifier was already stopping in a `t.Cleanup` above, so this invocation wasn't necessary.